src/ecalc/libraries/libecalc/common/tests/integration/test_consumer_system_v2.py

```diff
@@ -26,10 +30,34 @@ def result(consumer_system_v2_dto) -> EnergyCalculatorResult:
     return result
 
 
+parameterized_v2_parameters = [
+    (
+        "consumer_system_v2",
+        consumer_system_v2_dto(),
+    ),
+    (
+        "consumer_system_v2_with_temporal_operational_settings",
+        consumer_system_v2_dto_temporal_operational_settings(consumer_system_v2_dto()),
+    ),
+    (
+        "consumer_system_v2_dto_temporal_operational_settings_and_temporal_compressor_models",
+        consumer_system_v2_dto_temporal_operational_settings_and_temporal_compressor_models(consumer_system_v2_dto()),
+    ),
+    (
+        "consumer_system_v2_dto_with_overlapping_temporal_operational_settings_and_temporal_compressor_models",
+        consumer_system_v2_dto_with_overlapping_temporal_operational_settings_and_temporal_compressor_models(
+            consumer_system_v2_dto()
+        ),
+    ),
+]
+
+
+@pytest.mark.parametrize(
+    "name, consumer_system_v2",
+    parameterized_v2_parameters,
+)
 @pytest.mark.snapshot
-def test_compressor_system_v2_results(
-    result: EnergyCalculatorResult, rounded_snapshot, consumer_system_v2_dto: DTOCase
-):
+def test_compressor_system_v2_results(name: str, consumer_system_v2: DTOCase, request):
```

Comment:
I guess I would prefer to split the test, makes it easier to debug one of them. Or all in one as before, but separate tests and separate cases seems like a good idea